### PR TITLE
Updated possible heuristics

### LINKED ISSUE #14: Update possible heuristics
Currently, the [docs](https://www.neurobagel.org/documentation/dictionaries/#age) show possible heuristics to be `float, int, euro, bounded, range, iso8601` however, this is no longer the case see the relevant [PR](https://github.com/neurobagel/annotation_tool/pull/401) and current implementation [here](https://github.com/neurobagel/annotation_tool/blob/f29544e942d11c3fc90a20316f71de4b94ae9c06/store/index.js#L131-L140).

### PR DESCRIPTION
**Pull request by rmanaem**
Closes #14 

Changes proposed in this pull request:

- Updated possible heuristics

### CODE REVIEW

**Comment by surchs** [comment]
Thanks for the PR (and description, very helpful to retrace all these steps). I agree that the docs need updating. Removing `string` in https://github.com/neurobagel/annotation_tool/pull/401 made a lot of sense. I think it also made sense to remove `range` - because it is not clear to me what we should do with a range at the moment. However the [CLI currently still supports `range`](https://github.com/neurobagel/bagel-cli/blob/3890f2dd13430ddfb5d554c772a8b39a5e896705/bagel/pheno_utils.py#L20), so if we stick to not supporting `range`, it will have to be removed in the CLI as well (probably separate PR). @alyssadai: wdyt about getting rid of `range` across the board (until we have a better sense about what to do with it)?

**Comment by rmanaem** [comment]
@alyssadai I was able to trace it back to this [PR](https://github.com/neurobagel/annotation_tool/pull/368).

**Comment by surchs** [comment]
I think `iso8061` is a different problem than `range`: for `iso8061` we have a very clear idea of the workflow, we just haven't implemented the preview in the annotation tool yet (or so it seems), and therefore it lingers as commented out code. So the annotation tool needs an issue to enable the `iso8061` heuristic as a selectable option. Probably not the most urgent thing though at the moment, because we're not yet using the annotation tool to create the data dictionaries.

File: docs/dictionaries.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ this is a Neurobagel common data element.
 ## Age
 Neurobagel has a common data element for `"Age"` which describes a continuous column. To ensure age values are represented as floats in Neurobagel graphs, Neurobagel encodes the relevant "heuristic" describing the value format of a given age column. This heuristic, stored in the `Transformation` annotation, corresponds internally to a specific transformation that is used to convert the values to float ages.
 
-Possible heuristics are: `float`, `int`, `euro`, `bounded`, `range`, `iso8601`
+Possible heuristics are: `float`, `int`, `euro`, `bounded`
```
Comment (surchs) [inline docs/dictionaries.md:152]:
```suggestion
Possible heuristics are: `float`, `int`, `euro`, `bounded`, `iso8061`
```
Like we said in the discussion: `iso8061` is something we want, but don't have yet everywhere (i.e. the annotation tool). So let's keep it in the docs, but make an issue in the annotation_tool repo to make sure this heuristic gets added.

**Review by surchs — APPROVED** [review]
Thanks @rmanaem. Turns out there was something to discuss after all, so good job making an explicit PR! All looks good, but as discusssed, could you please accept suggestion and also make an annotation_tool issue to add `iso8061` back.

**Pull request merged**